### PR TITLE
MacOS 10.13 Software Updates Week 5

### PR DESCRIPTION
**Pull request by m-minasyan**
## Installed Software
- Java 1.8: (Zulu 8.44.0.11-CA-macosx) (build 1.8.0_242-b20) (default)
- NVM - Cached node versions: v13.7.0
### Package Management
- Homebrew 2.2.4
- RubyGems 3.1.2
### Project Management
- Git LFS: 2.10.0
- GNU parallel 20200122
#### PyPy
- 2.7.17
#### Xcode Support Tools
- Nomad CLI 3.1.2

### CODE REVIEW

File: images/macos/macos-10.13-Readme.md [hunk 1]
```diff
@@ -29,13 +29,14 @@ The following software is installed on machines in the Azure Pipelines **macOS-1
 - Bundler version 2.1.4
 - Carthage 0.34.0
 - CocoaPods 1.8.4
-- Homebrew 2.2.3
+- Homebrew 2.2.
```
Comment (miketimofeev) [inline images/macos/macos-10.13-Readme.md]: Something goes wrong here.
You don't need to change it manually, just copy software.md file

File: images/macos/macos-10.13-Readme.md [hunk 2]
```diff
@@ -77,7 +78,7 @@ The following software is installed on machines in the Azure Pipelines **macOS-1
 - 3.8.1
 
 #### PyPy (available through the [Use Python Version](https://go.microsoft.com/fwlink/?linkid=871498) task)
```
Comment (miketimofeev) [inline images/macos/macos-10.13-Readme.md]: we don't need `(available through the [Use Python Version](https://go.microsoft.com/fwlink/?linkid=871498) task)` string anymore

File: images/macos/macos-10.13-Readme.md (hunk 1 above)
Comment (m-minasyan) [reply images/macos/macos-10.13-Readme.md]: Done.

File: images/macos/macos-10.13-Readme.md (hunk 2 above)
Comment (m-minasyan) [reply images/macos/macos-10.13-Readme.md]: Done.

File: images/macos/macos-10.13-Readme.md [hunk 3]
```diff
@@ -11,31 +11,34 @@ The following software is installed on machines in the Azure Pipelines **macOS-1
...
 - Carthage 0.34.0
 - CocoaPods 1.8.4
-- Homebrew 2.2.3
+- Homebrew 2.2.4
 - NPM 3.10.10
 - Yarn 1.21.1
 - NuGet 4.7.0.5148
 - Pip 19.3.1 (python 2.7)
 - Pip 19.3.1 (python 3.7)
+- Rustup rustup:
```
Comment (AlenaSviridenko) [inline images/macos/macos-10.13-Readme.md]: please, remove this empty line

File: images/macos/macos-10.13-Readme.md [hunk 4]
```diff
@@ -11,31 +11,34 @@ The following software is installed on machines in the Azure Pipelines **macOS-1
...
 - Java 13: Zulu13.29+9-CA (build 13.0.2+6-MTS)
 - Node.js v6.17.0
 - NVM 0.33.11
-- NVM - Cached node versions: v6.17.1 v8.17.0 v10.18.1 v12.14.1 v13.6.0
+- NVM - Cached node versions: v6.17.1 v8.17.0 v10.18.1 v12.14.1 v13.7.0
 - PowerShell 6.2.3
 - Python 2.7.17
 - Python 3.7.6
 - Ruby 2.6.5p114
+- Rust rustc:
```
Comment (AlenaSviridenko) [inline images/macos/macos-10.13-Readme.md]: please, remove this empty line

Comment (miketimofeev) [inline images/macos/macos-10.13-Readme.md]: we don't need it

File: images/macos/macos-10.13-Readme.md (hunk 3 above)
Comment (miketimofeev) [inline images/macos/macos-10.13-Readme.md]: same

Comment (m-minasyan) [reply images/macos/macos-10.13-Readme.md]: Done.

File: images/macos/macos-10.13-Readme.md (hunk 4 above)
Comment (m-minasyan) [reply images/macos/macos-10.13-Readme.md]: Done.

Comment (m-minasyan) [reply images/macos/macos-10.13-Readme.md]: Done.

File: images/macos/macos-10.13-Readme.md (hunk 3 above)
Comment (m-minasyan) [reply images/macos/macos-10.13-Readme.md]: Done.

**Comment by m-minasyan** [comment]
@alepauly the deployment is finished - merge this one, please.

**Pull request merged**